src/library-authoring/utils.tsx

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/prefer-default-export */
+export const buildCollectionUsageKey = (libraryId: string, collectionId: string) => {
+  const orgLib = libraryId.replace('lib:', '');
+  return `lib-collection:${orgLib}:${collectionId}`;
+};
```

Comment:
Nit: I would prefer to put this in `src/generic/key-utils.ts` and give it a docstring.